pages/projects/[id]/branches.tsx

```diff
@@ -289,6 +289,17 @@ export const BranchesPage = ({ projects, currentProject }: Props) => {
             hasFilters={false}
             columns={protectedBranchesColumns}
             data={protectedBranches}
+            emptyStateProps={{
+              title: "No protected branches yet.",
+              icon: GitBranchPlus,
+              description: "You can start protecting your branches",
+              actionText: "from project settings page.",
+              onActionClick: () => {
+                router.push(
+                  `/project/${router.query.id}/settings/protected-branches`,
```

Comment:
Lets rename this to projectId so its easier to replace when @ttebify picks up #195
```suggestion
                  `/project/${projectId}/settings/protected-branches`,
```

Reply:
Okay